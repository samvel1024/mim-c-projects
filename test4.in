f4
e3
f5
g5
e3
