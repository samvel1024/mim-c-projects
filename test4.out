--------1
--------2
--------3
---BC---4
---CB---5
--------6
--------7
--------8
abcdefgh
? 0
--------1
--------2
--------3
---BC---4
---CB---5
--------6
--------7
--------8
abcdefgh
? 0
--------1
--------2
--------3
---BC---4
---CB---5
--------6
--------7
--------8
abcdefgh
f5 f4 0
--------1
--------2
--------3
---BBB--4
---CCC--5
--------6
--------7
--------8
abcdefgh
? 0
--------1
--------2
--------3
---BBB--4
---CCC--5
--------6
--------7
--------8
abcdefgh
e3 d6 -2
--------1
--------2
----C---3
---BCB--4
---BBC--5
---B----6
--------7
--------8
abcdefgh
