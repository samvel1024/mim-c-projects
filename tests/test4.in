4
-3 -3 -100 -2