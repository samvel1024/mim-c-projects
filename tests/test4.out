-3 3 -7 18 
